antismash/outputs/html/css/style.scss

```diff
@@ -1712,7 +1712,7 @@ ul.dropdown-options {
         * th, * td {
             padding: .8em 1em;
         }
-        thead tr v{
```

Comment:
Don't include unrelated changes in the same commit. This one in particular might have follow-on effects and it would be nice to know in the history when and why it was changed.